tests/test_resource.py

```diff
@@ -330,6 +334,31 @@ def test_options_request(self):
             ALLOW_ORIGIN_HEADER: ORIGIN_EXAMPLE,
         }
 
+    def test_cors_headers_param_request(self) -> None:
+        inst = self.make_cors_handler(req_origin=ORIGIN_EXAMPLE, cors_headers=["X-PRN-KEY"])
+        inst.method = "OPTIONS"
+        assert inst().headers == {
+            "Access-Control-Allow-Headers": (
+                "Content-Type, X-Amz-Date, Authentication, Authorization, X-Api-Key, "
+                "X-Amz-Security-Token, X-PRN-KEY"
+            ),
+            "Access-Control-Allow-Methods": "GET, POST, OPTIONS",
+            ALLOW_ORIGIN_HEADER: ORIGIN_EXAMPLE,
+        }
+
+    def test_cors_headers_env_request(self) -> None:
+        environ["CORS_HEADERS"] = "X-PRN-KEY,X-PRN-TOKEN"
```

Comment:
The env variable should be unset after the finished job, otherwise, it can affect the next tests 😉 
```python
@patch.dict(environ, {"CORS_HEADERS": "X-PRN-KEY,X-PRN-TOKEN"})
def test_cors_headers_env_request(self) -> None:
    ...
```